README.rst

```diff
@@ -1,11 +1,72 @@
-This package provides an ``ExplainCollection`` class
-that allows that allows PyMongo's Collection methods to be explained_
+==============
+PyMongoExplain
+==============
+
+:Info: Explain collections in PyMongo. See
+       `GitHub <https://github.com/mongodb-labs/pymongoexplain>`_
+       for the latest source.
+:Author: Julius Park
+
+About
+=====
+This package provides an ``ExplainCollection`` class that allows PyMongo's Collection methods to be explained_
 
 .. _explained: https://docs.mongodb.com/master/reference/command/explain/#dbcmd.explain.
 
 
+Support / Feedback
+==================
+
+For issues with, questions about, or feedback for PyMongoExplain, please look into
+our `support channels <http://www.mongodb.org/about/support>`_. Please
```

Comment:
I am not sure that we can mention our official support channels here. I don't believe this package will have the same level of support as the official driver. @ShaneHarvey @behackett thoughts?

Reply:
Agree. Let's just mention the community site:
```
For questions, discussions, or general technical support, visit the MongoDB Community Forums. The MongoDB Community Forums are a centralized place to connect with other MongoDB users, ask questions, and get answers.
```

Reply:
This sounds good to me.

Reply:
Done.